src/pages/components/code-block.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Code block
+description: >
+  Code block is used to display text with code formatting. It uses syntax
+  highlighting to make code more legible.
+package: '@zendeskgarden/react-typography'
+components:
+  - typography/src/elements/CodeBlock.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To display read-only text with code formatting</li>
+      <li>To display code snippets for users to copy</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>For editing code. Code snippets are read-only.</li>
+      <li>
+        <Markdown>
+          For simple, single-line usage, use the [Code](/components/code) component
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+Code block renders code with syntax highlighting.
+
+import Default from '../../examples/typography/code-block/Default.tsx';
+import DefaultCode from '!!raw-loader!../../examples/typography/code-block/Default.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Light mode
+
+By default, Code block is dark. Shed some light on it using `isLight`.
```

Comment:
Capitalization of "Code" but not "block" feels weird here. Either the whole thing is a proper noun ("Code Block") or it's not ("code block").

Reply:
@allisonacs this is the capitalization convention for multi-word components and would be a consistency update across many pages of the website. See
- https://garden.zendesk.com/components/button-group
- https://garden.zendesk.com/components/icon-button
- https://garden.zendesk.com/components/multi-thumb-range
- https://garden.zendesk.com/components/tooltip-modal
- etc